commands/rename_pipeline.go

```diff
@@ -0,0 +1,32 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/concourse/fly/rc"
+)
+
+type RenamePipelineCommand struct {
+	Pipeline string `short:"p"  long:"pipeline" required:"true"      description:"Pipeline to rename"`
```

Comment:
Maybe `-o` and `-n` for `--old-name` and `--new-name`. Using `-p` is consistent but here we have two pipeline names and it would be good to be explicit.